### PR TITLE
[to #134] Add test cases for cdc integration test

### LINKED ISSUE #134: cdc: Increase test coverage
### 1. Describe the bug

The test coverage is low (totally about 59.67% for master). We should increase it.
See https://app.codecov.io/gh/tikv/migration.
image

### 5. What is your migration tool and TiKV version? (Required)

- TiKV:
- TiKV CDC:
- TiKV BR:
- TiKV Online Bulk Load:

### PR DESCRIPTION
**Pull request by zeminzhou**
Signed-off-by: zeminzhou <zhouzemin@pingcap.com>

### What problem does this PR solve?
1. Add more test case for cdc integration test.
2. Fixed an issue where cdc would not change state to finish after sync is complete.
Issue Number: close #134

Problem Description: **TBD**

### What is changed and how does it work?
1. Add more test case.
2. Mark finish when sync is complete.

### Code changes

- Has exported function/method change

### Check List for Tests

This PR has been tested by at least one of the following methods:
- Unit test
- Integration test

### CODE REVIEW

**Comment by zeminzhou** [comment]
PTAL~@pingyu @haojinming

File: cdc/tests/integration_tests/availability/run.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+export DOWN_TIDB_PORT
+
+function prepare() {
+	rm -rf $WORK_DIR && mkdir -p $WORK_DIR
+
+	start_tidb_cluster --workdir $WORK_DIR
+
+	cd $WORK_DIR
+
+	run_cdc_cli changefeed create \
```
[the remark this answers is not in the document]
Comment (zeminzhou) [reply cdc/tests/integration_tests/availability/run.sh:23]: I add `sleep 10` to solve this problem

**Comment by zeminzhou** [comment]
To improve readability, I try to modify test cases  in `.unmodify` and use `git mv` for moving it to `integration test`, but `git mv` dose not work. So I keep it in `.unmodify` for reading and copy it to `integration test`. 
At last, I will remove `.unmodify`.

**Review by pingyu — APPROVED** [review]
LGTM~

**Review by haojinming — APPROVED** [review]
LGTM

**Pull request merged**